### PR TITLE
Minor fixes of docs and code comments for `PedAnovaImportanceEvaluator` 

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

Follow the coding style and enable the sphinx code block.

### CODE REVIEW

**Review by nabenabe0928 — APPROVED** [review]
Thank you for the fixes, LGTM!

**Comment by nabenabe0928** [comment]
@y0z Could you review this PR?

File: optuna/importance/_ped_anova/evaluator.py [hunk 1]
```diff
@@ -86,14 +86,14 @@ class PedAnovaImportanceEvaluator(BaseImportanceEvaluator):
     Args:
         baseline_quantile:
             Compute the importance of achieving top-`baseline_quantile` quantile objective value.
```
Comment (y0z) [inline optuna/importance/_ped_anova/evaluator.py:88]:
```suggestion
            Compute the importance of achieving top-``baseline_quantile`` quantile objective value.
```

**Review by y0z — COMMENTED** [review]
I leave a comment.

**Review by y0z — APPROVED** [review]
LGTM

**Pull request merged**